db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Medium.create({address: "The BookStore Theme, 4th Store Beside that building, USA",
+    call: "800 1234 5678", email: "info@bookstore.com",
+     timing: "Mon - Fri: 7am - 10pm\n​Saturday: 8am - 10pm\nSunday: 8am - 11pm"})
```

Comment:
usually you should add rails db:seed to travis and github actions

Reply:
done